pkg/cassandra/session.go

```
@@ -190,6 +190,18 @@ func isSelect(query string) bool {
 	return true
 }
 
+// keyspaceOf returns the keyspace part of a fully qualified name such as
+// "metrics.temperature". It returns an empty string when the name carries no
+// keyspace qualifier.
+func keyspaceOf(qualified string) string {
```

golangci-lint (2.12.2)

[error] func keyspaceOf is unused

(unused)